src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
+  MountService.MountService.registerComponent(
```

Comment:
Ahh yeah, meant to share the JIRA ticket: https://jira.mesosphere.com/browse/DCOS-13463